zram-generator.conf.example

```diff
@@ -6,14 +6,11 @@
 #
 # "host-memory-limit = none" may be used to disable this limit. This
 # is also the default.
-#
-# To disable a device (when some configuration for it already exits),
-# set "host-memory-limit = 0".
 host-memory-limit = 9048
 
 # The fraction of memory to use as ZRAM. For example, if the machine
 # has 1 GiB, and zram-fraction=0.25, then the zram device will have
-# 256 MiB. Values in the range 0.10–0.40 are recommended.
+# 256 MiB. Values in the range 0.10–0.50 are recommended.
 #
 # The default is 0.5.
 zram-fraction = 0.10
```

Comment:
Could you match defaults here and elsewhere in example config? I think this will be more intuitive than some random values and also more ready to use example config as current one without having to tweak it.

Reply:
The example config is much more simpler than this one: https://src.fedoraproject.org/rpms/rust-zram-generator/blob/master/f/zram-generator.conf

Reply:
Similarly, the config I bundle with my Debian package does actually match: https://github.com/nabijaczleweli/systemd-zram-generator.deb/blob/debian/debian/zram-generator.conf. Distributor's choice v0v

Reply:
My point was that the upstream example is filled with arbitrary values which are more or less far from optimal ones which some users may treat as ready to use recommendations. Debian version looks much better in this regard and I believe it would serve as upstream example better otherwise this may end up with every distro shipping their own example which would be a unnecessary duplication of work and waste.